src/ai/backend/gateway/logs.py

```diff
@@ -0,0 +1,203 @@
+from datetime import datetime
+import logging
+import uuid
+
+from aiohttp import web
+import aiohttp_cors
+import sqlalchemy as sa
+import trafaret as t
+from typing import Any, Tuple, MutableMapping
+
+from ai.backend.common import validators as tx
+from ai.backend.common.logging import BraceStyleAdapter
+
+from .auth import auth_required
+from .manager import READ_ALLOWED, server_status_required
+from .typing import CORSOptions, Iterable, WebMiddleware
+from .utils import check_api_params, get_access_key_scopes
+
+from ..manager.models import (
+    error_logs, LogSeverity, UserRole, groups,
+    association_groups_users as agus
+)
+
+log = BraceStyleAdapter(logging.getLogger('ai.backend.gateway.logs'))
+
+
+@server_status_required(READ_ALLOWED)
+@auth_required
+@check_api_params(t.Dict(
+    {
+        t.Key('severity'): tx.Enum(LogSeverity),
+        t.Key('source'): t.String,
+        t.Key('message'): t.String,
+        t.Key('context_lang'): t.String,
+        t.Key('context_env'): tx.JSONString,
+        t.Key('request_url', default=None): t.Null | t.String,
+        t.Key('request_status', default=None): t.Null | t.Int,
+        t.Key('traceback', default=None): t.Null | t.String
+    }
+))
+async def create(request: web.Request, params: Any) -> web.Response:
```

Comment:
It may not look important, but considering the nature of a logging system, it would be better to use names like "append".